crates/stdarch-gen/neon.spec

```diff
@@ -1339,3 +1339,121 @@ validate 8., 9., 10., 11., 12., 13., 14., 15.
 
 aarch64 = zip2
 generate float32x2_t, float32x4_t, float64x2_t
+
+////////////////////
+// Unsigned Absolute difference and Accumulate Long
+////////////////////
+
+/// Unsigned Absolute difference and Accumulate Long
+name = vabal
+multi_fn = vabd-unsigned-noext, b, c, d:in_t
+multi_fn = simd_add, a, {simd_cast, d}
+a = 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16
+b = 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16
+c = 10, 10, 10, 10, 10, 10, 10, 10, 20, 0, 2, 4, 6, 8, 10, 12
+validate 10, 10, 10, 10, 10, 10, 10, 10, 20, 20, 20, 20, 20, 20, 20, 20
+
+arm = add
+aarch64 = add
```

Comment:
This looks suspicious. Why are you checking for an `add` instruction instead of the SIMD `sabal`/`uabal` (AArch64) or `vabal` (ARM) instruction.

Reply:
> This looks suspicious. Why are you checking for an `add` instruction instead of the SIMD `sabal`/`uabal` (AArch64) or `vabal` (ARM) instruction.

Done. Thanks